docs/tech-stack/proxmox.md

```diff
@@ -32,7 +32,7 @@ You might be the right fit for virtualising via Proxmox if you want to run more
 
 ![proxmox-ui](../images/igpu-passthrough/image-5.png)
 
-I know this because since discovering just how powerful Intel's Quick Sync was, I rebuilt my PMS around this technology and wrote about it on [blog.ktz.me](https://blog.ktz.me/passthrough-intel-igpu-with-gvt-g-to-a-vm-and-use-it-with-plex/) and under *Advanced Configuration -> [Quick Sync and Proxmox passthrough](passthrough-igpu-gvtg) using Intel's GVT-g virtual GPU technology to pass more than a integrated graphics unit to more than one VM.
+I know this because since discovering just how powerful Intel's Quick Sync was, I rebuilt my PMS around this technology and wrote about it on [blog.ktz.me](https://blog.ktz.me/passthrough-intel-igpu-with-gvt-g-to-a-vm-and-use-it-with-plex/) and under *Advanced Configuration -> [Quick Sync and Proxmox passthrough](https://perfectmediaserver.com/advanced/passthrough-igpu-gvtg/) using Intel's GVT-g virtual GPU technology to pass more than a integrated graphics unit to more than one VM.
```

Comment:
Shoud use a relative link here, part so it's not tied directly to the domain, but also allows `mkdocs` to do some link checking.

```suggestion
I know this because since discovering just how powerful Intel's Quick Sync was, I rebuilt my PMS around this technology and wrote about it on [blog.ktz.me](https://blog.ktz.me/passthrough-intel-igpu-with-gvt-g-to-a-vm-and-use-it-with-plex/) and under *Advanced Configuration -> [Quick Sync and Proxmox passthrough](../advanced/passthrough-igpu-gvtg.md) using Intel's GVT-g virtual GPU technology to pass more than a integrated graphics unit to more than one VM.
```

(I _think_ this suggestion is the right URL)

Reply:
@RealOrangeOne knew you would make the link correct. Wasn't sure how was meant to do relative link